src/index.html

```diff
@@ -19,6 +19,21 @@
         <div class="col">
           <h1>Hello world</h1>
         </div>
+        <div class="col">
+          <form class="mt-4">
+            <input
+              class="form-control mb-4"
+              type="text"
+              placeholder="your email"
+            />
+            <input
+              class="form-control mb-4"
+              type="text"
+              placeholder="full name"
+            />
```

Comment:
How about we use a label element for each input element to improve the user experience? In this way we can have a specific label for each input such that users can focus the input by clicking the label.

Reply:
Great catch! I've added the labels for each input